gitlab.go

```diff
@@ -99,6 +99,9 @@ type Client struct {
 	// User agent used when communicating with the GitLab API.
 	UserAgent string
 
+	// Request options that are applied to every request to the GitLab API.
+	requestOptions []RequestOptionFunc
```

Comment:
```suggestion
	defaultRequestOptions []RequestOptionFunc
```
And please place this one above the `UserAgent`...